<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Chinese" country="China"/>
   </property>
   <widget class="QMenu" name="menu111">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="newpage"/>
    <addaction name="open"/>
    <addaction name="save"/>
    <addaction name="saveas"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_Y"/>
    <addaction name="action_X"/>
    <addaction name="action_C"/>
    <addaction name="action_V"/>
    <addaction name="action_D"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查看</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>缩放</string>
     </property>
     <addaction name="separator"/>
     <addaction name="action_16"/>
     <addaction name="action_17"/>
     <addaction name="action_18"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="separator"/>
    <addaction name="action_20"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu111"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="newpage">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="open">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>保存
</string>
   </property>
  </action>
  <action name="saveas">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_Y">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="action_V">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="action_D">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>恢复默认缩放</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>状态栏
</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>help</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>说明</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>版本</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
